egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Pegah Ghahremani
+#
+# Apache 2.0.
+
+""" This script generates egs.archive.scp and ranges.* used for multilingual setup.
+    Also this script generates outputs.*.ark and weight.*.ark, where each line
+    corresponds to language-id and weight for the same example in egs.*.scp.
+    weight.*.ark is archive of table indexed by the key of input example
```

Comment:
Hm, this program seems to only generate temporary output so I guess I'm OK with those names.
But that was not at all obvious from the comment at the top, and some of that text I gave you was wrong for that case.  What I want is for the comment at the top to clearly and correctly explain what is going on.  Can you please try to do that, and then check it with Vimal before asking me to re-review?